src/cinaps/dune

```diff
@@ -0,0 +1,4 @@
+(library
+ (name ppxlib_cinaps_helpers)
+ (public_name ppxlib.cinaps_helpers)
```

Comment:
Why is this library public?

Reply:
Hah! Sorry, I was just used to writing dune files that way.